plugin/src/com/microsoft/alm/plugin/external/visualstudio/VisualStudioTfvcCommands.java

```diff
@@ -30,6 +32,8 @@
     private static final Pattern WORKFOLD_REPORT_PATTERN = Pattern.compile("^(.*?): (.*)$");
     private static final Pattern WORKFOLD_WORKSPACE_NAME_AND_OWNER_PATTERN = Pattern.compile("^(.*?) \\((.*)\\)$");
 
+    private static final String AUTHENTICATION_ERROR_PREFIX = "TF30063:";
```

Comment:
If we wanted to handle other errors in future? If yes we should create enumeration or class for error code container and parser?

Reply:
No, I don't think we want to handle any other errors in the future. Our interaction with Visual Studio TFVC client is very limited (we only get the projects' details to import them into our client), and there're no plans to significantly expand it.